public/app/plugins/datasource/influxdb/specs/query_part.jest.ts

```diff
@@ -1,4 +1,5 @@
 import queryPart from '../query_part';
+import { registerAngularDirectives } from '../../../../core/core';
```

Comment:
This needs to be removed since it's not used. That's why the frontend build failed.